velox/dwio/common/Options.h

```diff
@@ -72,7 +72,34 @@ enum class SerDeSeparator {
 
 class SerDeOptions {
  public:
+  /// The following members control how data is separated TEXT file:
+  ///
+  /// - 'separators': An array of separator characters used to delimit columns
+  ///   and nested data.
+  ///     - 'separators[0]' separates top-level columns.
+  ///     - 'separators[depth_]' separates nested ComplexType columns.
```

Comment:
///    - 'separators[0]' defines the delimiter that separates top-level columns.
  ///     - 'separators[1 to depth_-1]' defines the delimiters that separate nested data within a ComplexType column.